aes/cipher.go

```diff
@@ -172,7 +171,7 @@ func (c Cipher) Encrypt(plaintext interface{}, key []byte, additionalData string
 	case bool:
 		encryptedType = "bool"
 		// The Python version encodes booleans with Titlecase
-		plainBytes = []byte(strings.Title(strconv.FormatBool(value)))
+		plainBytes = []byte(fmt.Sprintf("%t", value))
```

Comment:
Isn't this a behavior change? Before it was resulting in `True` and `False`, now it results in `true` and `false`.

Since [strings.Title](https://pkg.go.dev/strings#Title) is deprecated, how about writing this out?
```suggestion
		if (value) {
			plainBytes = []byte("True")
		} else {
			plainBytes = []byte("False")
		}
```
?

Reply:
You are absolutely right.